apps/web/src/hooks/useDomain.ts

```diff
@@ -1,11 +1,25 @@
+import { ChainId } from '@pancakeswap/sdk'
 import { useSidNameForAddress } from 'hooks/useSid'
 import { useUnsNameForAddress } from 'hooks/useUns'
 import { useMemo } from 'react'
+import { useEnsName } from 'wagmi'
+import useActiveWeb3React from './useActiveWeb3React'
 
-export const useDomainNameForAddress = (address: string, fetchData = true) => {
+export const useDomainNameForAddress = (address: `0x${string}` | string, fetchData = true) => {
+  const { chainId } = useActiveWeb3React()
   const { sidName, isLoading: isSidLoading } = useSidNameForAddress(address, fetchData)
   const { unsName, isLoading: isUnsLoading } = useUnsNameForAddress(address, fetchData && !sidName && !isSidLoading)
+  const { data: ensName, isLoading: isEnsLoading } = useEnsName({
+    address: address as `0x${string}`,
+    chainId,
+    cacheTime: 3_600,
+    enabled: chainId !== ChainId.BSC && chainId !== ChainId.BSC_TESTNET,
+    scopeKey: address,
```

Comment:
shouldn't need it?

Reply:
Do you mean scopeKey? 
yeah, maybe we don't need it